src/types/Options.ts

```diff
@@ -1,9 +1,15 @@
 import { DefaultEnvVars, UndefinedEnvVars } from './EnvVars';
 import { RemoveKeys } from './helpers';
 
+export interface EnvOptions {
+  pathToEnv?: string;
+  envFileName?: string;
+}
```

Comment:
We should probably make this opt-in for now, since if other projects are using `dotenv` or similar files this may overwrite what those projects do. Let's add an option `loadDotEnv?: boolean` to enable it. When we go to version 2 of this project we can make it the default along with any other breaking changes.